plugins/commands/autocomplete/command/root.rb

```diff
@@ -0,0 +1,64 @@
+require "optparse"
+require 'vagrant/util/install_cli_autocomplete'
+
+module VagrantPlugins
+  module CommandAutocomplete
+    module Command
+      class Root < Vagrant.plugin("2", :command)
+        def self.synopsis
+          "manages autocomplete: installation, removal, etc."
```

Comment:
Maybe we should only say "installation on host" or something until we get a remove sub command :thinking: Thoughts?